vars/env/prod/env.yml

```diff
@@ -84,8 +84,8 @@ env_host_url: "{{ env_external_dns_name }}"
 
 # django environment specific apps list
 # A test client - moved to aws-test / dev /impl settings
-env_django_environment_apps: ['storages', ]
-env_django_optional_installed_apps: [ ]
+env_django_environment_apps: ['storages', 'apps.testclient', ]
+env_django_optional_installed_apps: ["testclient", ]
```

Comment:
should it be apps.testclient on line 88

Reply:
TEST and IMPL values are the exact same way.  Not sure it matters anymore if we are going to remove the optional apps loading.

Reply:
I had the same thought, but on TEST and IMPL, the values are exactly like this so I thought it best to leave it alone unless there is an explanation for change?